### PR TITLE
Refactor the DevTools installation

### PR DESCRIPTION
**Pull request by j-f1**

### CODE REVIEW

**Comment by j-f1** [comment]
❓

File: app/src/lib/devtools-installer.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    [REACT_DEVELOPER_TOOLS, REACT_PERF].map(id => installExtension(id))
+  )
+
+  ready
+    .then(names =>
+      console.log(`[devtools-installer] Installed: ${names.join(', ')}`)
+    )
+    .catch(err =>
+      console.error(
+        `[devtools-installer] Failed to install devtools extension:\n${err.stack}`
```
Comment (iAmWillShepherd) [inline app/src/lib/devtools-installer.ts]: This should be sufficient.
`console.error("[devtools-installer] Failed to install devtools extension", err)`

**Comment by iAmWillShepherd** [comment]
@j-f1 were you still intending to get your changes in or can I close this PR?

**Comment by j-f1** [comment]
I wanted to get these changes in — It would be really useful to have the different inspectors immediately available.

Closing + reopening to kick the CIs.

**Comment by j-f1** [comment]
Any idea why this is happening?

```
  App
    1) opens a window on launch
  0 passing (3s)
  1 failing
  1) App opens a window on launch:
     Error: Invalid Chai property: browserWindow
      at Object.proxyGetter [as get] (node_modules/chai/lib/chai/utils/proxify.js:66:17)
      at Context.<anonymous> (app/test/integration/launch-test.ts:39:13)
      at process._tickDomainCallback (internal/process/next_tick.js:135:7)
```

**Comment by shiftkey** [comment]
@j-f1 try running this locally to see the same problem: `npm run build:prod && npm run test:integration`

**Comment by j-f1** [comment]
This isn’t that important, and I’m not sure what broke the tests.

**Pull request closed without merging**